quickstarts/jenkins-prometheus/config.yml

```
@@ -0,0 +1,87 @@
+id: 980ab07a-3483-4b44-bbac-ed9c0a9a41a1
```

The system will automatically generate an id for new quickstarts, please remove this line
```suggestion
id: 980ab07a-3483-4b44-bbac-ed9c0a9a41a1
```